server/src/main/java/keywhiz/auth/bcrypt/BcryptAuthenticator.java

```diff
@@ -48,7 +52,7 @@ public BcryptAuthenticator(UserDAO userDAO) {
     String password = credentials.getPassword();
 
     // Get hashed password column from BCrypt table by username
-    Optional<String> optionalHashedPwForUser = userDAO.getHashedPassword(username);
+    Optional<String> optionalHashedPwForUser = userDAO.getHashedPassword(dslContext, username);
```

Comment:
I didn't even address `SecretController.java` which makes things worse. I'm very unclear how we should move forward.